x-pack/docs/en/rest-api/security/roles.asciidoc

```diff
@@ -1,205 +1,9 @@
-{
-  "role": {
-    "created": true <1>
-  }
-}
---------------------------------------------------
-// TESTRESPONSE
-<1> When an existing role is updated, `created` is set to false.
-
-[[security-api-get-role]]
-To retrieve a role from the `native` Security realm, issue a GET request to the
-`/_xpack/security/role/<rolename>` endpoint:
-
-[source,js]
---------------------------------------------------
-GET /_xpack/security/role/my_admin_role
---------------------------------------------------
-// CONSOLE
-// TEST[continued]
-
-A successful call returns an array of roles with the JSON representation of the
-role. If the role is not defined in the `native` realm, the request 404s.
-
-[source,js]
---------------------------------------------------
-{
-  "my_admin_role": {
-    "cluster" : [ "all" ],
-    "indices" : [ {
-      "names" : [ "index1", "index2" ],
-      "privileges" : [ "all" ],
-      "field_security" : {
-        "grant" : [ "title", "body" ]
-      },
-      "query" : "{\"match\": {\"title\": \"foo\"}}"
-    } ],
-    "applications" : [ ],
-    "run_as" : [ "other_user" ],
-    "metadata" : {
-      "version" : 1
-    },
-    "transient_metadata": {
-      "enabled": true
-    }
-  }
-}
---------------------------------------------------
-// TESTRESPONSE
-
-You can specify multiple roles as a comma-separated list. To retrieve all roles,
-omit the role name.
-
-[source,js]
---------------------------------------------------
-# Retrieve roles "r1", "r2", and "my_admin_role"
-GET /_xpack/security/role/r1,r2,my_admin_role
-
-# Retrieve all roles
-GET /_xpack/security/role
---------------------------------------------------
-// CONSOLE
-// TEST[continued]
-
-NOTE: If single role is requested, that role is returned as the response. When 
-requesting multiple roles, an object is returned holding the found roles, each 
-keyed by the relevant role name.
-
-[[security-api-delete-role]]
-To delete a role, submit a DELETE request to the `/_xpack/security/role/<rolename>`
-endpoint:
-
-[source,js]
---------------------------------------------------
-DELETE /_xpack/security/role/my_admin_role
---------------------------------------------------
-// CONSOLE
-// TEST[continued]
-
-If the role is successfully deleted, the request returns `{"found": true}`.
-Otherwise, `found` is set to false.
-
-[source,js]
---------------------------------------------------
-{
-  "found" : true
-}
---------------------------------------------------
-// TESTRESPONSE
-
-[[security-api-clear-role-cache]]
-The Clear Roles Cache API evicts roles from the native role cache. To clear the
-cache for a role, submit a POST request `/_xpack/security/role/<rolename>/_clear_cache`
-endpoint:
-
-[source,js]
---------------------------------------------------
-POST /_xpack/security/role/my_admin_role/_clear_cache
---------------------------------------------------
-// CONSOLE
+* <<security-api-put-role,Create role>>, <<security-api-delete-role,Delete role>>
```

Comment:
Should delete be on its own line/bullet?

Reply:
I've been generally pairing them, but we can re-style if necessary once all the changes are in.